sdk/storage/storage-blob/test/appendblobclient.spec.ts

```diff
@@ -63,4 +65,140 @@ describe("AppendBlobClient", () => {
     assert.equal(await bodyToString(downloadResponse, content.length), content);
     assert.equal(downloadResponse.contentLength!, content.length);
   });
+
+  it("can be created with a url and a credential", async () => {
+    const factories = appendBlobClient.pipeline.factories;
+    const credential = factories[factories.length - 1] as SharedKeyCredential;
+    const newClient = new AppendBlobClient(appendBlobClient.url, credential);
+
+    await newClient.create();
+    await newClient.download();
+  });
+
+  it("can be created with a url and a credential and an option bag", async () => {
+    const factories = appendBlobClient.pipeline.factories;
+    const credential = factories[factories.length - 1] as SharedKeyCredential;
+    const newClient = new AppendBlobClient(appendBlobClient.url, credential, {
+      telemetry: { value: "test/1.0" }
+    });
+
+    await newClient.create();
+    await newClient.download();
+  });
+
+  it("can be created with a url and a pipeline", async () => {
+    const factories = appendBlobClient.pipeline.factories;
+    const credential = factories[factories.length - 1] as SharedKeyCredential;
+    const pipeline = newPipeline(credential);
+    const newClient = new AppendBlobClient(appendBlobClient.url, pipeline);
+
+    await newClient.create();
+    await newClient.download();
+  });
+
+  it("can be created with a connection string", async () => {
+    const newClient = new AppendBlobClient(
+      process.env.CONNECTION_STRING || "",
+      containerName,
+      blobName
+    );
+
+    await newClient.create();
+    await newClient.download();
+  });
+
+  it("throws error when passed an invalid connection string", async () => {
+    try {
+      const newClient = new AppendBlobClient(
+        "DefaultEndpointsProtocol=a;AccountName=b;AccountKey=c;EndpointSuffix=d" || "",
+        containerName,
+        blobName
+      );
+
+      await newClient.create();
```

Comment:
need a `assert.fail()` at the end of `try` block, and assert that we catch an error in `catch` block.